### PR TITLE
PROD30376: Upgrade Drupal to 10.3

### PR DESCRIPTION
**Pull request by vcsvinicius**
## Problem
Upgrade Drupal to 10.3 version

## Solution
- Replace deprecated function
- Remove and replace old patches
- Update Behat test to be compatible with Drupal 10.3
- Upgrade Drupal 10.3

## Issue tracker
[PROD-30376](https://getopensocial.atlassian.net/browse/PROD-30376)

## Theme issue tracker
N/A

## How to test
- [ ] Regression test

## Screenshots
N/A

## Release notes
Drupal release notes:
- https://www.drupal.org/project/drupal/releases/10.3.0-beta1
- https://www.drupal.org/project/drupal/releases/10.3.0-rc1
- https://www.drupal.org/project/drupal/releases/10.3.0
- https://www.drupal.org/project/drupal/releases/10.3.1
- https://www.drupal.org/project/drupal/releases/10.3.2

## Change Record
N/A

## Translations
N/A

[PROD-30376]: https://getopensocial.atlassian.net/browse/PROD-30376?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

File: modules/custom/activity_send/src/Plugin/SendActivityDestinationBase.php [hunk 1]
```diff
@@ -192,8 +192,16 @@ public static function getSendTargetUsers(Activity $activity) {
...
+   * @throws \Exception
    */
-  public static function isUserOffline(User $account) {
+  public static function isUserOffline(User $account): bool {
+    // When the session table doesn't exist, the user is off.
+    $session_exist = \Drupal::database()->schema()->tableExists('sessions');
+    if (!$session_exist && PHP_SAPI === 'cli') {
+      return TRUE;
+    }
+
```
Comment (ronaldtebrake) [inline modules/custom/activity_send/src/Plugin/SendActivityDestinationBase.php:204]: Nice one!

File: modules/custom/mentions/src/Plugin/Filter/MentionsFilter.php [hunk 2]
```diff
@@ -275,7 +275,7 @@ public function filterMentions(string $text): string {
             '#render_value' => $output['value'],
             '#render_plain' => $output['render_plain'] ?? FALSE,
           ];
-          $mentions = $this->renderer->renderPlain($build);
+          $mentions = $this->renderer->renderInIsolation($build);
```
Comment (ronaldtebrake) [inline modules/custom/mentions/src/Plugin/Filter/MentionsFilter.php:278]: Considering this lands in our 13 alpha, it's not an issue but thought I'd share anyway.
What we could've done is become forward compatible, this means we're compatible with both 10.2 and 10.3

Using

```
/** @var \Drupal\Core\Render\RendererInterface $renderer */
$renderer = \Drupal::service('renderer');
$output = DeprecationHelper::backwardsCompatibleCall(
  currentVersion: \Drupal::VERSION,
  deprecatedVersion: '10.3',
  currentCallable: fn() => $renderer->renderInIsolation($elements),
  deprecatedCallable: fn() => $renderer->renderPlain($elements),
);```

Basically now we have to require Drupal 10.3 as our only accepted core version. 
Again that is fine, but in the future we might want to becompatible with multiple versions of core and the above is the way to do it.

File: phpstan-baseline.neon [hunk 3]
```diff
@@ -2425,11 +2425,6 @@ parameters:
 			count: 1
 			path: modules/custom/social_language/src/Routing/RouteSubscriber.php
 
-		-
-			message: "#^Method Drupal\\\\social_language\\\\SocialLanguageConfigOverride\\:\\:createConfigObject\\(\\) should return Drupal\\\\Core\\\\Config\\\\StorableConfigBase but returns null\\.$#"
-			count: 1
-			path: modules/custom/social_language/src/SocialLanguageConfigOverride.php
-
```
Comment (ronaldtebrake) [inline phpstan-baseline.neon:2432]: Nice cleanups!

File: modules/social_features/social_branding/tests/src/Kernel/GraphQL/BrandingCommunityFieldsPresenceGraphQLTest.php [hunk 4]
```diff
@@ -8,7 +8,7 @@
 /**
  * Base class for branding related GraphQL tests.
  */
-class BrandingGraphQLTest extends SocialGraphQLTestBase {
```
Comment (ronaldtebrake) [inline modules/social_features/social_branding/tests/src/Kernel/GraphQL/BrandingCommunityFieldsPresenceGraphQLTest.php:11]: Awesome effort in splitting this up 👍

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

File: modules/custom/activity_send/modules/activity_send_email/src/Plugin/EmailFrequency/Immediately.php [hunk 5]
```diff
@@ -174,7 +174,7 @@ protected function sendEmail(string $body_text, string $langcode, User $target,
     ];
 
     // Construct the body & subject for email sending.
-    $params['body'] = $this->renderer->renderPlain($notification);
+    $params['body'] = $this->renderer->renderInIsolation($notification);
```
Comment (ribel) [inline modules/custom/activity_send/modules/activity_send_email/src/Plugin/EmailFrequency/Immediately.php:177]: Hi @vcsvinicius,
Please note that we just merged some fixes to this file, so you might need to update it. See here: https://github.com/goalgorilla/open_social/pull/4092/files

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Review by robertragas — APPROVED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by BiaInacio** [comment]
regression task done, all looks good!

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr4029-ce2aqbnzrhnxxoxn4thbwepch4l3rvxn.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/66bd030db4d70ade375041f8

**Pull request merged**